lib_nbgl/src/nbgl_layout.c

```diff
@@ -726,9 +792,9 @@ static nbgl_container_t *addContentCenter(nbgl_layoutInternal_t      *layoutInt,
 
     // get container children
     container->nbChildren = 0;
-    container->children   = nbgl_containerPoolGet(5, layoutInt->layer);
+    container->children   = nbgl_containerPoolGet(6, layoutInt->layer);
```

Comment:
Could be interesting to have defines instead of hardcoded values. Here, it is difficult to understand

Reply:
I'll add some details about why 6 :)